values/aws-alb-ingress-controller/aws-alb-ingress-controller-raw.gotmpl

```diff
@@ -12,11 +12,5 @@ resources:
         alb.ingress.kubernetes.io/scheme: internet-facing
         alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80},{"HTTPS":443}]'
         alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
-        {{- if getOrNil $v.cluster "ingress.allowedCidrs" }}
```

Comment:
The PR is about keycloak as an IdP. Therefor I do not understand why aws-alb-ingress-controller configuration is changed.